settings.py

```diff
@@ -0,0 +1,12 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "City temperature management API"
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
```

Comment:
The `DATABASE_URL` is set to use SQLite with `aiosqlite`. Ensure that this is the intended database for your application. If you plan to use a different database in production, update this URL accordingly.